app/styles/application.scss

```diff
@@ -30,3 +31,43 @@ $govuk-font-family: Frutiger W01, arial, sans-serif;
 .govuk-body {
   line-height: 1.47368;
 }
+
+.app-prose-scope {
+  // Simulate design used in NHS design system
+  figure {
+    @include govuk-responsive-margin(6, "bottom");
+    @include govuk-responsive-margin(6, "top");
+
+    background-color: govuk-colour("white");
+    border-bottom: 1px solid #d8dde0;
+  }
+
+  figcaption {
+    @include govuk-typography-responsive(16);
+    color: $govuk-text-colour;
+    margin-top: 0;
+    padding: govuk-spacing(3);
+
+    * {
+      @include govuk-typography-responsive(16);
+    }
+
+    :last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  // Improve display of mission patch images inside figures
+  figure:has(img[src$="mission-patch.png"]) {
```

Comment:
Bit of a hack but it'll do for now. We could add a separate page of all the mission patches at some point?

Reply:
Oh don’t worry, that’s on my list!

Reply:
Mine too!